src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIActionPlanEventServiceTest.kt

```diff
@@ -50,6 +53,25 @@ class CommunityAPIActionPlanEventServiceTest {
     )
   }
 
+  @Test
+  fun `notify approved action plan`() {
+
+    val event = getEvent(APPROVED)
+    communityAPIService.onApplicationEvent(event)
+
+    verify(communityAPIClient).makeAsyncPostRequest(
+      "/secure/offenders/crn/X123456/sentence/1234/notifications/context/commissioned-rehabilitation-services",
+      NotificationCreateRequestDTO(
+        "ACC",
+        sentAtDefault,
+        event.actionPlan.referral.id,
+        approvedAtDefault,
+        "Action Plan Approved for Accommodation Referral XX1234 with Prime Provider Harmony Living\n" +
```

Comment:
_nit_
From what I've seen, content is usually preferred to not have camel case, eg.
```suggestion
        "Action plan approved for Accommodation referral XX1234 with prime provider Harmony Living\n" +
```